libs/util/app-config/src/lib/app-config.ts

```diff
@@ -195,6 +195,54 @@ export function loadAppConfig() {
           : ({
               FILTER_GEOMETRY_DATA: parsedSearchSection.filter_geometry_data,
               FILTER_GEOMETRY_URL: parsedSearchSection.filter_geometry_url,
+              CUSTOM_FILTERS: (parsedSearchSection.custom_filters as [])?.map(
+                (filter) => {
+                  switch (filter) {
+                    case 'organisation':
+                      return {
+                        fieldName: 'publisher',
```

Comment:
we shouldn't do any mapping with field names here, only the fields service has this power :)